onnxruntime/test/onnx/main.cc

```diff
@@ -472,6 +469,8 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"bitshift_left_uint32", "BitShift(11) not implemented yet"},
       {"bitshift_left_uint16", "BitShift(11) not implemented yet"},
       {"gemm_default_scalar_bias", "Gemm ValidBroadcast() has bug to be fixed."},
+      {"reflect_pad", "test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests"},
+      {"edge_pad", "test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests"},
 };
```

Comment:
Is there a reason why we shouldn't enable int32_t support if it was felt to be important enough to validate in the onnx tests?

Reply:
Well again this is controversial -

The feedback at the time of the ONNX change was that since we have multiple backend tests for this op (multiple modes), it does make sense to have more variance in terms of the types that we were testing for.

On the flip side, there has never been a need to warrant supporting `int32_t` (this op has been around since opset-2 and no model using ORT has required `int32_t`). Is covering ONNX backend tests a good enough reason to warrant support in the kernel (and bloat up binary size) ? I don't know...

Reply:
For now - let's support an `int32_t` kernel when there is a real model requirement and remove the tests from exclusion then ?